### ISSUE TITLE
Build error on Fedora kernel 6.6

### ISSUE THREAD

**Issue by agates** [pstolarz/w1-gpio-cl#6]
Running Fedora on kernel 6.6 and I'm getting the following error:

```
# /usr/bin/gmake all NUM_THREADS=4
NOTE: ./w1 symlink is already set and will not be updated. Remove it and restart the compilation process in case you want to re-run the kernel sources examination.
NOTE: gen-mast.h was generated
/usr/bin/gmake -C /lib/modules/6.6.14-200.fc39.aarch64/build M=/usr/local/src/w1-gpio-cl modules
gmake[1]: Entering directory '/usr/src/kernels/6.6.14-200.fc39.aarch64'
  CC [M]  /usr/local/src/w1-gpio-cl/w1-gpio-cl.o
/usr/local/src/w1-gpio-cl/w1-gpio-cl.c: In function ‘init_module’:
/usr/local/src/w1-gpio-cl/w1-gpio-cl.c:497:46: error: ‘GPIOF_OPEN_DRAIN’ undeclared (first use in this function)
  497 |                         GPIOF_IN | (mdt.od ? GPIOF_OPEN_DRAIN : 0),
      |                                   
```

Looks like `GPIOF_OPEN_DRAIN` was [removed in 6.5](https://github.com/torvalds/linux/commit/9df8c63c2b814be8c40d44d21dabaf074058c98b)

**Comment by pstolarz** [comment]
If you comment out this line and compile the module, does it work properly?

**Comment by pstolarz** [comment]
Fixed by 50af4d514415fefe432e80e4d7a1802c404af205
